ksp_plugin_adapter/flight_planner.cs

```diff
@@ -258,64 +258,42 @@ class FlightPlanner : VesselSupervisedWindowRenderer {
         final_times.Add(double.PositiveInfinity);
 
         for (int i = 0; i < burn_editors_.Count; ++i) {
+          Style.HorizontalLine();
+          // In the future, coast information, e.g., orbit analysis will be
+          // shown between these lines.  For now, we only have a button to
+          // insert a burn.
+          if (RenderAddManœuvreButton(i)) {
+            return;
```

Comment:
You cannot exit these loops at random places, it would create layout/repaint inconsistencies.

Let's take most of the C# out of this PR and just do the minimal changes to make things compile.  A lot more care is required to make the UI code work.

Reply:
Nevermind.